### ISSUE TITLE
Cannot read property 'items' of undefined

### ISSUE THREAD

**Issue by wienzzz** [eakoriakin/ionic-select-searchable#59]
Hi man! awesome component, However I am running to one issue here. If I type non-existent item name on filter (eg: zzz) ionic will display error with message. Typing existent item name is fine though. 

```
TypeError: Cannot read property 'items' of undefined
    at Object.eval [as updateDirectives] (ng:///SelectSearchableModule/SelectSearchablePageComponent.ngfactory.js:481:48)
    at Object.debugUpdateDirectives [as updateDirectives] (http://localhost:8100/build/vendor.js:15072:21)
    at checkAndUpdateView (http://localhost:8100/build/vendor.js:14219:14)
    at callViewAction (http://localhost:8100/build/vendor.js:14570:21)
    at execEmbeddedViewsAction (http://localhost:8100/build/vendor.js:14528:17)
    at checkAndUpdateView (http://localhost:8100/build/vendor.js:14220:5)
    at callViewAction (http://localhost:8100/build/vendor.js:14570:21)
    at execComponentViewsAction (http://localhost:8100/build/vendor.js:14502:13)
    at checkAndUpdateView (http://localhost:8100/build/vendor.js:14225:5)
    at callViewAction (http://localhost:8100/build/vendor.js:14570:21)
```

Some snippet of my html code 

```
<ion-item>
                  <select-searchable formControlName="agen"
                      [items]="_agents"
                      itemValueField="id_agen"
                      itemTextField="nm_agen"
                      [hasVirtualScroll]="true"
                      [canSearch]="true"
                      (onChange)="agentChange($event)">
                      <ng-template selectSearchableLabelTemplate>
                          <ion-icon name="person"></ion-icon>
                      </ng-template>
                  </select-searchable>
              </ion-item>
```

and in TS 

```
interface IAgen {
  id_agen: string;
  nm_agen: string;
  alamat: string;
  no_telp: string;
  tipe: boolean;
  flagact: boolean;
  create_time: any;
  last_update: any;
}

@IonicPage()
@Component({
  selector: 'page-transaksi',
  templateUrl: 'transaksi.html',
})
export class TransaksiPage {
  _agent: IAgen;
  _agents: IAgen[];
  private transaksi_init: FormGroup;
  constructor(public alert:AlertController, public load:LoadingController, public myData: DataProvider, public navCtrl: NavController, public navParams: NavParams, private formBuilder: FormBuilder) {
    this.transaksi_init = this.formBuilder.group({
      tipe: [1, Validators.required],
      nota: ['', Validators.required],
      agen: ['', Validators.required],
      tanggal: [''],
    });

    this.myData.getListAgen().then((data: any) => {
      this._agents = data.data;
    });
  }
```
I can't figure out what's wrong, but I know I am certainly wrong since in your demo typing up non-existent item will retrun no items found.
Can you give me any hint about this error?

**Comment by eakoriakin** [comment]
Hi, try to set a default empty value for `items`:
```
export class TransaksiPage {
    _agents: IAgen[] = [];
}
```

**Comment by wienzzz** [comment]
@eakoriakin not working

I try to reproduce it to new page

1. ionic g page testsearch 

1. import SelectSearchableModule on testsearch.module.ts

1. testsearch.html file
```
<!--
  Generated template for the TestsearchPage page.

  See http://ionicframework.com/docs/components/#navigation for more info on
  Ionic pages and navigation.
-->
<ion-header>

  <ion-navbar>
    <ion-title>testsearch</ion-title>
  </ion-navbar>

</ion-header>


<ion-content padding>
    <ion-item>
        <select-searchable
      [(ngModel)]="port"
      itemValueField="id"
      itemTextField="name"
      [items]="ports"
      [hasVirtualScroll]="true"
      [canSearch]="true"
      (onChange)="portChange($event)">
      <ng-template selectSearchableLabelTemplate>
        Port
      </ng-template>
    </select-searchable>
    </ion-item>
</ion-content>
```

4. testsearch.ts
```
import { Component } from '@angular/core';
import { IonicPage, NavController, NavParams } from 'ionic-angular';
import { SelectSearchableComponent } from 'ionic-select-searchable';

/**
 * Generated class for the TestsearchPage page.
 *
 * See https://ionicframework.com/docs/components/#navigation for more info on
 * Ionic pages and navigation.
 */

@IonicPage()
@Component({
  selector: 'page-testsearch',
  templateUrl: 'testsearch.html',
})
export class TestsearchPage {
  port: any;
  ports: any[] = [
    { id: 14, name: 'Cabo San Antonio' },
    { id: 15, name: 'Diamante' },
    { id: 16, name: 'San Antonio Este Arg' },
    { id: 44, name: 'Santa Cruz' }
  ];
  constructor(public navCtrl: NavController, public navParams: NavParams) {
  }

  portChange(event: {
    component: SelectSearchableComponent,
    value: any
  }) {
    console.log('port:', event.value);
  }
}
```
typing z would filter the item to Santa Cruz, but typing 'zz' would return same error Cannot read property 'items' of undefined.

my package.json file shows that I am using version 2.5.0
```
    "@angular/animations": "5.2.10",
    "@angular/common": "5.2.10",
    "@angular/compiler": "5.2.10",
    "@angular/compiler-cli": "5.2.10",
    "@angular/core": "5.2.10",
    "@angular/forms": "5.2.10",
    "@angular/http": "5.2.10",
    "@angular/platform-browser": "5.2.10",
    "@angular/platform-browser-dynamic": "5.2.10",
    "@ionic-native/core": "4.7.0",
    "@ionic-native/splash-screen": "4.7.0",
    "@ionic-native/status-bar": "4.7.0",
    "@ionic/storage": "2.1.3",
    "ionic-angular": "3.9.2",
    "ionic-select-searchable": "^2.5.0",
```

**Comment by eakoriakin** [comment]
Ok, let me check it

**Comment by eakoriakin** [comment]
I tried your sample and it works without errors. Can you share a git repo with this problem, so I could reproduce the error?

**Comment by eakoriakin** [comment]
Ok, I was able to reproduce it, will fix it

**Comment by wienzzz** [comment]
Thanks @eakoriakin. Waiting for the commit

**Comment by eakoriakin** [comment]
I fixed the bug in `ionic-select-searchable@2.5.1`.

**Comment by wienzzz** [comment]
I can confirm that error has been fixed on this version. Thanks @eakoriakin
